src/components/Search/Search.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import Lupa from '../../images/lupa.png'
+// import Icon from '../../images/2-2-anime-png-pic.png'
+
+import './Search.scss'
+
+const Search = ({type = 'search', placeholder}) => (
+  <div className='kkk'>
+     {/* <img className="icon" src={Icon} alt="icon" /> */}
+      <h1 className="welcome">, fulano</h1>
+      <p className='subtitle'>Let's watch together</p>
```

Comment:
We should use double quotes in HTML
```suggestion
      <p className="subtitle">Let's watch together</p>
```